pkg/dogstatsd/batch.go

```diff
@@ -55,7 +55,15 @@ func newBatcher(demux aggregator.Demultiplexer) *batcher {
 
 	_, pipelineCount := aggregator.GetDogStatsDWorkerAndPipelineCount()
 
-	e, sc := agg.GetBufferedChannels()
+	var e chan []*metrics.Event
+	var sc chan []*metrics.ServiceCheck
+
+	// the Serverless Agent is not running an Aggregator, it does not
+	// have to support service checks nor events.
```

Comment:
A small nit: I might be reading it wrong, but to me it sounds like the cause and effect are swapped in this sentence. To me it reads as serverless agent does not support checks and events, because it does not run an aggregator.